middleware/checkAnyUserExist.js

```diff
@@ -0,0 +1,20 @@
+const User = require("../models/User");
+const { createDefaultUser } = require("./createDefaultUser");
+const logger = require("../logger/dev-logger");
+
+const checkAnyUserExist = async (next) => {
+    try {
+        const userCount = await User.find().count();
+        if (userCount === 0) {
+            const result = await createDefaultUser();
+            console.log(`username: ${result.username}`)
+            console.log(`password: ${result.password}`)
+        }
+    } catch (error) {
+        next(error);
+    }
```

Comment:
https://dev.to/dev_tycodez/seed-mongodb-mongoose-seed-51d5

Reply:
Please tell me what exactly I need to do